src/main/java/org/entando/plugins/pda/pam/service/task/model/KieTaskSummaryResponse.java

```diff
@@ -0,0 +1,81 @@
+package org.entando.plugins.pda.pam.service.task.model;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.databind.annotation.JsonDeserialize;
+import edu.umd.cs.findbugs.annotations.SuppressFBWarnings;
+import java.util.Date;
+import lombok.Getter;
+import lombok.Setter;
+import org.entando.plugins.pda.pam.service.task.KieDateDeserializer;
+import org.kie.server.api.model.instance.TaskSummary;
+import org.springframework.beans.BeanUtils;
+
+@Getter
+@Setter
+@SuppressFBWarnings({"EI_EXPOSE_REP", "EI_EXPOSE_REP2"})
+@SuppressWarnings("PMD.TooManyFields")
+public class KieTaskSummaryResponse {
```

Comment:
You can use TaskSummary from the KIE Client library, it's the exact same object

Reply:
I needed to add the JsonProperty annotations, like this one `@JsonProperty("task-id")`. On their side they use a different parser.